lib/run/summary.js

```diff
@@ -2,7 +2,11 @@ var _ = require('lodash'),
     SerialiseError = require('serialised-error'),
 
     proxyPropertyList,
-    RunSummary;
+    RunSummary,
+    exportReference = {
```

Comment:
I would, but that requires shifting to a common format for globals and environment variables.